prisma/schema.prisma

```diff
@@ -94,6 +94,7 @@ model SetupMetadata {
   configured    Boolean   @default(false)
   initializedAt DateTime  @default(now())
   configuredAt  DateTime?
+  allowAnonymousRecruitment Boolean   @default(false)
```

Comment:
Just wondering if this model should now be called something like "AppSettings" rather than SetupMetadata. It seems to make more sense.